builtin/mainmenu/common.lua

```diff
@@ -300,3 +300,34 @@ function is_server_protocol_compat_or_error(proto_min, proto_max)
 
 	return true
 end
+
+--------------------------------------------------------------------------------
+local function menu_load_worldmt(selected)
```

Comment:
also, two lines is hardly duplication. Just replace the two calls to menu_load_worldmt() with the menudata line.